nautobot/extras/jobs.py

```diff
@@ -1144,20 +1144,23 @@ def run_job(self, job_class_path, *args, **kwargs):
         raise
 
 
-def enqueue_job_hooks(object_change):
+def enqueue_job_hooks(object_change, may_reload_jobs=True):
     """
-    Find job hook(s) assigned to this changed object type + action and enqueue them
-    to be processed
+    Find job hook(s) assigned to this changed object type + action and enqueue them to be processed.
+
+    Returns:
+        jobs_reloaded (bool): whether Jobs were reloaded to make this happen
     """
+    jobs_reloaded = False
 
     # Job hooks cannot trigger other job hooks
     if object_change.change_context == ObjectChangeEventContextChoices.CONTEXT_JOB_HOOK:
-        return
+        return jobs_reloaded
 
```

Comment:
It's a bit redundant but I was hesitant to remove it since (for whatever reason) we expose `enqueue_job_hooks` in the `nautobot.apps.jobs` namespace so in theory it could be called directly, bypassing the other check.